.github/labeler.yml

```diff
@@ -1,3 +1,3 @@
 ---
 skip-changelog:
- - head-branch: ['^release-*', 'release']
+ - head-branch: ['^release-*']
```

Comment:
why did you remove the `release` branch?

Reply:
This is a pattern. Not a name. And if somehow Release ist in the name. The label would be applied